.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: julia
 os:
     - osx
+      osx_image: xcode7
```

Comment:
@staticfloat do you also get that panicky feeling when you accidentally push a commit with a typo to a random branch and then trying to do a `git --amend, git push -f` as fast as possible before Tony catches it :P.